src/styles/content/tables/styles.less

```diff
@@ -103,9 +103,6 @@
 
     th,
     td {
-      // This property is necessary for table cells to respect the text-overflow
-      // property. http://stackoverflow.com/questions/9789723/css-text-overflow-in-a-table-cell
-      max-width: 0;
```

Comment:
This caused a different issue on the table for me.

![image](https://user-images.githubusercontent.com/10524/42458358-16170d36-839a-11e8-9a8f-3fdc5637670a.png)

Reply:
Sorry, I actually meant to add the `max-width` back

Reply:
Ok. I will look again tomorrow.